source_code/src/UTILS/delays.c

```diff
@@ -23,13 +23,19 @@
 *    Author:   Mathieu Stephan
 */ 
 #include "timer_manager.h"
+#include "mini_inputs.h"
 
 /*! \fn     userViewDelay(void)
 *   \brief  2secs delay for the user to view information
 */
 void userViewDelay(void)
 {
     timerBasedDelayMs(2000);
+
+#ifdef MINI_VERSION
+    // Discard user wheel input
+    miniWheelClearDetections();
```

Comment:
let's try to not put that kind of functions in delays.c
miniWheelClearDetections() is actually called in the critical places.
Please let us know which functions call userViewDelay which would require miniWheelClearDetections after its return.